### PR TITLE
FAD-6046 Redirect enterprise tenants to the summary report on login

### PR DESCRIPTION
**Pull request by bizob2828**
TODO:

- [x] create A-E review to add splashPage to the tenant config and set it to /dashboard for spc

### CODE REVIEW

File: src/config/routes.js [hunk 1]
```diff
@@ -140,19 +141,19 @@ const routes = [
   {
     path: '/account/subaccounts',
     component: subaccounts.ListPage,
-    condition: hasGrants('subaccount/manage'),
+    condition: hasGrants('subaccount/manage', 'api_keys/manage', 'sending_domains/manage'),
```
Comment (rajumsys) [inline src/config/routes.js]: what's going on here?

**Comment by rajumsys** [comment]
This looks good so far.
- Why more capabilities added to some of route (in `hasGrants`)? 
- If we're on dashboard, reloading (after updating `splashPage`) on redirect anywhere. This is not a problem as it won't happen in real life; just wanted to mention as it may happen during test. 
- are you going to do those A-E thing in this ticket? If so, let me know when that's ready for review.

File: src/config/routes.js (hunk 1 above)
Comment (bizob2828) [reply src/config/routes.js]: these were missing. you need all 3 grants for this page to properly load

**Comment by bizob2828** [comment]
Added A-D/A-E changes here https://fisheye.int.messagesystems.com/cru/APP-3103

**Pull request closed without merging**